LSP.sublime-settings

```diff
@@ -43,13 +43,79 @@
   // Use "clients" to override settings instead!
```

Comment:
Maybe this comment can be expanded a bit more. Have a link to the relevant documentation, or maybe explain why there's a default_clients and clients dictionary.

Reply:
There is already an explanation for the [clients](https://github.com/tomv564/LSP/blob/f088272c8dcdae51a22175853aab51654a759075/LSP.sublime-settings#L89-L92).

I just added a link to both the clients and [defaul_clients](https://github.com/tomv564/LSP/blob/f088272c8dcdae51a22175853aab51654a759075/LSP.sublime-settings#L97-L100) settings.

Is this good? :)